Design.md

```diff
@@ -229,6 +229,14 @@ This API will enable or disable user on Vault.
 ### Create S3 Credential
 This API creates S3 credential for the user.
 1. `generate-access-key` api will be called using assumed role credentials.
+1. Encrypt the `secret-key` using Base64 encoder and store the encrypted `secret-key` on Redis Cache with key formatted as `<Username>__<AccessKeyID>`.
+
```

Comment:
Can we store them in a sub-set ? to avoid polluting the global namespace ?

Reply:
Sure we can store all the [hash-key, value] pairs in a hash named `osis` to avoid any confusion

Reply:
Addressed in 9e7c162b06f6f686a847a767ce61219fd4caaf9f